test-utils/src/main/scala/com/basho/riak/spark/rdd/AbstractRiakTest.scala

```diff
@@ -41,6 +41,7 @@ abstract class AbstractRiakTest extends RiakFunctions{
   protected val DEFAULT_NAMESPACE = new Namespace("default","test-bucket")
   protected val DEFAULT_NAMESPACE_4STORE = new Namespace("default", "test-bucket-4store")
   protected val DEFAULT_RIAK_HOST = System.getProperty("com.basho.riak.pbchost", RiakNode.Builder.DEFAULT_REMOTE_ADDRESS)
+  protected val RIAK_KV_HOST = System.getProperty("com.basho.riak-kv.pbchost")
```

Comment:
To highlight necessity of providing KV port it is better to rename to "PROVIDED_RIAKKV_HOST"